test/models/match_test.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class MatchTest < ActiveSupport::TestCase
+  def test_match_must_belong_to_same_organization_as_pet
+    org1 = Organization.create!
+    org2 = Organization.create!
+    pet = Pet.create!(organization: org1,
+      name: "Harry", age: 5, size: "small", breed: "corgi", sex: "m", description: "test pet")
+    match = Match.create(pet: pet, organization_id: org2.id,
+      adopter_account: adopter_accounts(:adopter_account_one))
```

Comment:
I prefer to create test data in the test instead of using fixtures and I think we discussed doing that project wide. Ideally we could use something like factory_bot to fill in the irrelevant details of the model creation that aren't related to the test (in this case everything that isn't organization related). Unfortunately I didn't want to create all the stuff around `adopter_account` so I just reused the fixture in that case

Reply:
I agree that this kind of setup for tests is preferable.